test/external/mlperf_unet3d/kits19.py

```diff
@@ -0,0 +1,165 @@
+# https://github.com/mlcommons/training/blob/master/image_segmentation/pytorch/data_loading/pytorch_loader.py
```

Comment:
if this is a whole copy, update the link to a permalink, commented so and don't change anything (include indent)